plugins/samplemimo/bladerf2mimo/bladerf2mithread.cpp

```diff
@@ -17,9 +17,9 @@
 
 #include <algorithm>
 
-#include "bladerf2/devicebladerf2shared.h"
```

Comment:
You should replace it with "bladerf2/devicebladerf2.h" here and not add "bladerf2mimo.h" later

Reply:
done